.azure-pipelines/template-commonlib.yml

```diff
@@ -39,9 +39,10 @@ jobs:
       displayName: Make configure
     - script: |
         set -ex
+        grep amd64 .arch && extarget=target/docker-sonic-vs.gz
```

Comment:
docker-sonic-vs.gz should be only in one of the BLDENV.

Reply:
If split docker-sonic-vs.gz into another pipeline, it is complex to refer it. We need to add another step to download.
If they are in the same pipeline, we only need to add patterns in old download step.

Reply:
We can see this test job: https://dev.azure.com/mssonic/build/_build/results?buildId=107333&view=results
Although it is in buster, it is ok to run in bullseye. It will do nothing.